server/db/043-add-number-of-alerts-to-sessions.sql

```diff
@@ -0,0 +1,37 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 43;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        -- ADD SCRIPT HERE
```

Comment:
Check out how `035-add-session-responded-by-phone-number.sql` disables and re-enables the `set_sessions_timestamp` trigger. This makes it so that the `updated_at` value isn't automatically changed when you make updates. 

This would probably be useful in this case otherwise all the sessions will look as though they changed at the time when this script is run. What do you think?

Reply:
Oh! That's smart!!! I'll do this.